NEWS

```diff
@@ -44,17 +44,34 @@ issues]].
   - Fixed a contraction for word "кое-где", now it is contracted
     according to rules.
   - Other minor fixes and improvements.
+Updated the Danish tables to the new 2022 Danish Braille standard
+  Thanks to Bue Vester-Andersen.
+  - Limited contraction level (grade 2l or 1.5)is no longer part of the Danish
+    Braille standard 2022.
+  - There is no longer any need for "literary" tables. In Danish Braille,
+    capital letters are now always indicated, and steps have been
+    taken in the standard to ensure better back-translation.
+  - As requested by The Danish Braille Committee, the tables for the
+    1993 standard will be kept as an option for at least a year.
+    All the 1993 tables have been renamed to include "_1993" in the
+    filename, so the new tables will be used as default	
```

Comment:
Braille is be spelled with a lowercase "b" (except when in certain names such as "The Danish Braille Committee". We try to consistently do this correctly. I can change it.